document/CoreFormat.mdk

```diff
@@ -141,19 +141,38 @@ An `sarifLog` object specifies the version of the file format and contains the o
 
 ### `version` property { #sarifLog-version }
 
-A `sarifLog` object shall contain a property named `version` whose value is the string `"1.0.0-beta.2"`.
+A `sarifLog` object shall contain a property named `version` whose value is a string designating the
+version of the SARIF format to which this log file conforms.
+This string must have the value `"1.0.0"`.
+
+~ IsoNote
+The value of this property, in the version of SARIF described by
+this document and in all future versions of the SARIF standard, will be in the format specified by Semantic
+Versioning 2.0.0.
+~
 
 Although the order in which the name/value pairs appear in a JSON object value is not semantically significant,
 the `version` property should appear first. 
+
 ~ IsoNote
 This will make it easier for parsers to handle multiple versions of the SARIF format,
 if new versions are defined in the future.
 ~
 
+### `$schema` property { #sarifLog-dollarSchema }
+
+A `sarifLog` object may contain a property named `$schema` whose value is a string containing a
+URI from which a JSON schema describing the version of the SARIF format to which this log file conforms
+can be obtained.
+
+If the `$schema` property is present, the JSON schema obtained from the specified URI must describe
```

Comment:
I don't think so. The value of $schema is "a URI I can go to get the schema for the currrent version". There's no guarantee that the URI ends with the version number (although it would be a good idea). I can't specify how the owners of schema stores must assign their URIs.

---
In reply to: [63398590](https://github.com/sarif-standard/sarif-spec/pull/181#discussion_r63398590) [](ancestors = 63398590)